.github/workflows/pr-checklist.yml

```diff
@@ -1,8 +1,8 @@
-name: Add editorial checklist upon review
+name: Add editorial checklist upon PR
 
 on:
-  pull_request_review:
-    types: [submitted]
+  pull_request:
+    branches: [ master, main ]
```

Comment:
This just gets triggered now upon commit instead of  

```
 pull_request_review:
    types: [submitted]
```

Was this on pupose?